### PR TITLE
feat : auth added

### PR DESCRIPTION
**Pull request by arpit01923**
-Auth

1. Auth contains Login, Logout and signup page

-What files to check :

1. src/pages/authentication/Login.jsx
2. src/pages/authentication/Logout.jsx
3. src/pages/authentication/Signup.jsx

-Preview
image
image
image

### CODE REVIEW

File: src/pages/authentication/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          Password
+          <input
+            class="inp"
+            type="password"
+            placeholder="Min. 8 characters"
+            required
+          />
+        </label>
+        <input class="submit" type="submit" name="" />
+      </form>
```
Comment (Aditi0505) [inline src/pages/authentication/Login.jsx:25]: To avoid repetition of code, you can create an Input component and pass the values as props to it.

**Review by Aditi0505 — APPROVED** [review]
Left one suggestion. 
1. The folder structure seems good. 
2. The UI is really clean.
3. Code is easily readable. Just the repetition can be avoided at some places.

File: src/pages/authentication/Logout.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import "styles/authentication/Logout.css"
+
+const Logout = () => (
+  <>
```
Comment (SanketDhabarde) [inline src/pages/authentication/Logout.jsx]: The is one common section. so this fragment is not needed

File: src/pages/authentication/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import "styles/authentication/Signup.css";
+
+const loginData = ["Firstname", "Lastname", "Email", "Password"];
+const Signup = () => (
+  <>
```
Comment (SanketDhabarde) [inline src/pages/authentication/Signup.jsx]: Same here

File: src/pages/authentication/Login.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import "styles/authentication/Login.css";
+
+const Login = () => (
+  <>
```
Comment (SanketDhabarde) [inline src/pages/authentication/Login.jsx]: Same here no need

File: src/pages/authentication/Login.jsx [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import "styles/authentication/Login.css";
+
+const Login = () => (
```
Comment (SanketDhabarde) [inline src/pages/authentication/Login.jsx:5]: if there is only one component you want to export you can directly do `export const Login = () => {}`

File: src/pages/authentication/Login.jsx (hunk 1 above)
Comment (SanketDhabarde) [reply src/pages/authentication/Login.jsx:34]: +1

**Review by SanketDhabarde — APPROVED** [review]
Left a few suggestions.
Everything else looks good. The code looks clean and readable.

File: src/pages/authentication/Login.jsx (hunk 5 above)
Comment (arpit01923) [reply src/pages/authentication/Login.jsx:5]: to main consistency I would export it later

**Pull request merged**